From 2276fb95cff97375ca38b6901480a94f8c8806cc Mon Sep 17 00:00:00 2001
From: Weilun Du <wdu@google.com>
Date: Fri, 22 Jun 2018 13:48:21 -0700
Subject: [PATCH 09/81] Copy dummy userdata.img to sdk_addon images.

Avd manager in Android Studio requires userdata.img in the downloaded
system image. As we no longer inlcude the real userdata.img, we copy the
dummy userdata.img to pass the check.
This cl does not impact real devices.

BUG: 110492064
Test: Manual
1, Build with command "make -j110 PRODUCT-sdk_gphone_x86-sdk_addon".
2, Check if the dummy userdata.img appears in sdk addon image zip.
Change-Id: Icb88aac42921502288efe7355b8cd76704d5eb30
Signed-off-by: Weilun Du <wdu@google.com>
---
 core/tasks/sdk-addon.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/tasks/sdk-addon.mk b/core/tasks/sdk-addon.mk
index 2a20f070e..55a55d4bb 100644
--- a/core/tasks/sdk-addon.mk
+++ b/core/tasks/sdk-addon.mk
@@ -67,6 +67,7 @@ files_to_copy += \
 	$(addon_dir_img):$(INSTALLED_QEMU_VENDORIMAGE):images/$(TARGET_CPU_ABI)/vendor.img \
 	$(addon_dir_img):$(BUILT_RAMDISK_TARGET):images/$(TARGET_CPU_ABI)/ramdisk.img \
 	$(addon_dir_img):$(PRODUCT_OUT)/system/build.prop:images/$(TARGET_CPU_ABI)/build.prop \
+	$(addon_dir_img):device/generic/goldfish/data/etc/userdata.img:images/$(TARGET_CPU_ADI)/userdata.img \
 	$(addon_dir_img):$(target_notice_file_txt):images/$(TARGET_CPU_ABI)/NOTICE.txt \
 	$(addon_dir_img):$(PRODUCTS.$(INTERNAL_PRODUCT).PRODUCT_SDK_ADDON_SYS_IMG_SOURCE_PROP):images/source.properties
 
-- 
2.17.1

